### ISSUE TITLE
Feature request: add 'recent projects' history in the open project menu

### ISSUE THREAD

**Issue by afrigeri** [andrea-bistacchi/PZero#11]
Adding recent projects (Open recent project ->) history in the menu so that the last project can be opened quickly.

**Comment by gbene** [comment]
Yes, this would be very convenient. We could approach this by creating a hidden text file (eg .history) where a fixed amount (5-10?) of file paths are saved (in chronological order). Every time pzero is launched the .history file is read and the info displayed by filling the Open recent project sub menu.
It shouldn't be too hard. What do you think?

p.s. the .history file needs to be registered in the .gitignore to avoid possible errors

**Comment by andrea-bistacchi** [comment]
To make this a general solution that will work also with methods we could use in the future to distribute and run pzero (e.g. pyinstaller) we need (1) to put this file in a standard directory such as c:/user/<your user>/appdata/pzero (but this will depend on the OS), and (2) to let pzero create an empty file at startup if it does not find it.

**Comment by afrigeri** [comment]
to follow up the .history and dedicated local conf dir idea, a solid solution would be to introduce a pzerco.conf file to be read with [configparser](https://docs.python.org/3/library/configparser.html).  This way we can incorporate the 5-10 history file list, introducing a config file for future needs.

**Comment by andrea-bistacchi** [comment]
... this is similar to a .ini file in windows, that can be created in the pzero root directory the first time it runs, whether or not from a pyinstaller installation, and added to gitignore for the development version.

Correct?

**Comment by afrigeri** [comment]
correct.  We can use something like this:
```
[history]
project_files = [ '/path/to/file.p0' ,  '/path/to/another/file.p0' , ... ]
```
and in the future, we can add specific sections if/when needed:
```
[default_dirs] 
temp_dir   = '/tmp/' 
export_dir = '...' 
```
for this specific #11, the ```.history``` file from @gbene is clean and quick to implement and minimizes complexity at this stage (and maybe we move to configopts as soon as we foresee configuration needs)
